manual/qmcpack_papers.bib

```diff
@@ -0,0 +1,1676 @@
+  author={Vincent, Jordan Eric},
+  year={2006},
+  school={University of Illinois at Urbana-Champaign},
+  url={http://hdl.handle.net/2142/80541}
+}
+
+% (thesis) gergely water
+@phdthesis{gergely2009thesis,
+  title={Quantum Monte Carlo Methods for First Principles Simulation of Liquid Water},
+  author={Gergely, John Robert},
+  year={2009},
+  school={University of Illinois at Urbana-Champaign},
+  url={http://hdl.handle.net/2142/80610}
+}
+
+% (thesis) clark order N
+@phdthesis{clark2009thesis,
+  title={Strongly Correlated Systems Approach Through Quantum Monte Carlo},
+  author={Clark, Bryan},
+  year={2009},
+  school={University of Illinois at Urbana-Champaign},
+  url={http://hdl.handle.net/2142/80605},
+}
+
+% (thesis) beaudet hydrogen on carbon/transition metal
+@phdthesis{beaudet2011thesis,
+  title={Quantum Monte Carlo Study of Hydrogen Adsorption on Carbon and Transition Metal Systems},
+  author={Beaudet, Todd D},
+  year={2011},
+  school={University of Illinois at Urbana-Champaign},
+  url={http://hdl.handle.net/2142/18600}
+}
+
+% (thesis) ahuja linear scaling qmc
+@phdthesis{ahuja2011thesis,
+  title={Recycling Krylov subspaces and preconditioners},
+  author={Ahuja, Kapil},
+  year={2011},
+  school={Virginia Polytechnic Institute and State University},
+  url={http://theses.lib.vt.edu/theses/available/etd-11112011-010340/}
+}
+
+% (thesis) krogel germanium energy density
+@phdthesis{krogel2013thesis,
+  title={Energetics of neutral interstitials in germanium},
+  author={Krogel, Jaron},
+  year={2013},
+  school={University of Illinois at Urbana-Champaign},
+  url={http://hdl.handle.net/2142/44417},
+}
+
+% (thesis) mcminis pressure entanglement
+@phdthesis{mcminis2013thesis,
+  title={Benchmark studies using quantum Monte Carlo: pressure estimators, energy, and entanglement},
+  author={McMinis, Jeremy},
+  year={2013},
+  school={University of Illinois at Urbana-Champaign},
+  url={http://hdl.handle.net/2142/44344},
+}
+
+% (thesis) niu global array bspline orbitals
+@phdthesis{niu2015thesis,
+  title={Characterization and Enhancement of Data Locality and Load Balancing for Irregular Applications},
+  author={Niu, Qingpeng},
+  year={2015},
+  school={The Ohio State University},
+  url={https://etd.ohiolink.edu/!etd.send_file?accession=osu1420811652&disposition=attachment}
+}
+
+% (thesis) deible multidet molecules
+@misc{deible2015thesis,
+           month = {September},
+           title = {Theory and Applications of Quantum Monte Carlo},
+          author = {Michael Deible},
+            year = {2015},
+        keywords = {Quantum Monte Carlo},
+             url = {http://d-scholarship.pitt.edu/25747/},
+}
+
+% (thesis) yang molecule entanglement
+@phdthesis{yang2016thesis,
+  title={Pairing and entanglement: quantum Monte Carlo studies},
+  author={Yang, David Chang-Mo},
+  year={2016},
+  school={University of Illinois at Urbana-Champaign},
+  url={http://hdl.handle.net/2142/90737},
+}
+
+
+
+
+
+% cite qmcpack only
```

Comment:
Anything below this point either only cites QMCPACK or is a derivative work that does not use QMCPACK.  These should be excluded from the citation list (though tracking them separately is possibly a matter of internal interest).

Reply:
Removed